public/index.html

```diff
@@ -26,9 +26,9 @@
     -->
     <title>React App</title>
   </head>
-  <body>
+  <body style="min-height: 100vh; height: 100vh;">
```

Comment:
Let's declare this css inside the `index.css` file, instead of using inline styling.